hudi-integ-test/prepare_integration_suite.sh

```diff
@@ -42,7 +42,7 @@ get_spark_command() {
   else
     scala=$scala
   fi
-  echo "spark-submit --packages org.apache.spark:spark-avro_${scala}:2.4.4 \
+  echo "spark-submit --packages org.apache.spark:spark-avro_${scala}:3.1.3 \
```

Comment:
`--packages org.apache.spark:spark-avro_${scala}:3.1.3 \` is no longer needed.  We should delete that.

Reply:
will remove